engine/src/main/java/org/hibernate/search/engine/spi/EntityIndexBinding.java

```diff
@@ -50,6 +50,16 @@
 	void postInitialize(Set<Class<?>> indexedClasses);
 
 	/**
+	 * @return the {@code IndexManager} impl used by this entity type
+	 */
+	Class<? extends IndexManager> getIndexManagerImpl();
```

Comment:
What's the difference between this method and `IndexManager[] getIndexManagers();`?
Shouldn't this be called `getIndexManager()`?